packages/smoke-test/test/commands/__snapshots__/fix.test.ts.snap

```diff
@@ -723,6 +725,27 @@ export default class HelloWorld extends Component<HelloWorldCommentSignature> {
 "
 `;
 
+exports[`fix command > ember_js_app_4.11 --mode=single-pass 7`] = `"{{yield (hash foo=\\"bar\\")}}"`;
+
+exports[`fix command > ember_js_app_4.11 --mode=single-pass 8`] = `
+"import Component from '@glimmer/component';
+
+export default class AudienceDemographics extends Component {
+  /**
+   * List of items to be rendered in the bar chart
+   * @typedef {{ count: number }} HeadCount
+   * @returns {HeadCount}
+   */
+  // @ts-expect-error @rehearsal TODO TS2304: Cannot find name 'HeadCount'.
+  get something(): HeadCount {
```

Comment:
Is this the expected result? We should not create the interface because it won't be created safely?

Reply:
For now we decided do not create type if it's not it the right place (if created type not in the SourceFile) to unblock migration. The next idea is to, probably, modify codefix result to put type in the root / namespace / function body.

+ https://github.com/microsoft/TypeScript/issues/55308